doc/api/crypto.md

```diff
@@ -2057,6 +2081,68 @@ Based on the recommendations of [NIST SP 800-131A][]:
 
 See the reference for other recommendations and details.
 
+### CCM mode
+
+CCM is one of the two supported [AEAD algorithms][]. Applications which use this
+mode must adhere to certain restrictions when using the cipher API:
+
+- The authentication tag length must be specified during cipher creation by
+  setting the `authTagLength` option.
+- The length of the initialization vector (nonce) must be one of 4, 6, 8, 10,
+  12, 14, or 16 bytes.
+- When decrypting, the authentication tag must be set via `setAuthTag()` before
+  specifying additional authenticated data and / or calling `update()`.
+  Otherwise, decryption will fail and `final()` will throw an error. Note that
```

Comment:
I think we should note that using CCM as a stream is not recommended in CCM mode.
```
It is not recommended to use stream processing such as `write(data)`, `end(data)` or `pipe()` with CCM mode. 
```